infrastructure/scripts/provision-infra.sh

```
@@ -0,0 +1,117 @@
+else
+    TAG="${TIMESTAMP}"
+fi
+
+echo "Using image tag: $TAG"
+
+ENVIRONMENT="${ENVIRONMENT:-Local}"
+DJANGO_CONFIGURATION="${DJANGO_CONFIGURATION:-Local}"
+DJANGO_SETTINGS_MODULE="${DJANGO_SETTINGS_MODULE:-settings.local}"
+DOMAIN_NAME="${DOMAIN_NAME:-localhost}"
+ENABLE_CRON_TASKS="${ENABLE_CRON_TASKS:-false}"
++DB_PASSWORD="${DB_PASSWORD:-$(openssl rand -base64 24)}"
```

Shellcheck (0.11.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)